api/session_tracker.py

```
@@ -0,0 +1,558 @@
+        select(SshSession).where(
+            SshSession.ended_at.is_(None),
+            SshSession.username == event["username"],
+            SshSession.source_ip == event["ip"],
+            SshSession.source_port == event["port"],
+        )
+    )
+    if existing.scalars().first() is not None:
+        return
+
+    result = await db.execute(
+        select(User).where(User.username == event["username"], User.is_active == True)
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)